isthmus/src/main/java/io/substrait/isthmus/sql/SubstraitSqlToCalcite.java

```diff
@@ -41,6 +42,23 @@ public static RelRoot convertQuery(String sqlStatement, Prepare.CatalogReader ca
     return convertQuery(sqlStatement, catalogReader, validator, createDefaultRelOptCluster());
   }
 
+  /**
+   * Converts a SQL statement to a Calcite {@link RelRoot}.
+   *
+   * @param sqlStatement a SQL statement string
+   * @param catalogReader the {@link Prepare.CatalogReader} for finding tables/views referenced in
+   *     the SQL statement
+   * @param operatorTable the {@link SqlOperatorTable} for dynamic operators
```

Comment:
Strictly speaking, you can inject a SqlOperatorTable here that incorporates dynamic operators, but you can also inject a SqlOperatorTable that doesn't. What this allows you to do is have more fine-grained control over valid operators by allowing you to control what SqlOperatorTable you use, instead of just being forced to use the default SubstraitOperatorTable that we provide.

```suggestion
   * @param operatorTable the {@link SqlOperatorTable} for controlling valid operators
```